### DISCUSSION TITLE
Is it possible to use only one endpoint as Edge Function

### DISCUSSION THREAD

**Discussion by ugurkellecioglu** [nuxt/nuxt#20323] [Questions]
I have a Nuxt 3 app that I deployed on Vercel. I'm wondering if there's a way to deploy only one of my endpoints as an edge function, while the rest remain as same. I tried configuring it with 'runtime: "edge"' but it didn't work. It affects to whole project.

I only need one endpoint to be an edge function

**Comment by zernonia** [comment]
I believe this is being provision in [Nitro](https://github.com/unjs/nitro/issues/1158) recently
